docs/changelog.md

```diff
@@ -1,3 +1,18 @@
+v0.28 + 1
+---------
+
+### Changes or improvements
+
+* The CMake option to use a system http-parser library, instead of the
+  bundled dependency, has changed.  This is due to a deficiency in
+  http-parser that we have fixed in our implementation.  The bundled
+  library is now the default, but if you wish to force the use of the
+  system http-parser implementation despite incompatibilities, you can
+  specify `-DUSE_BUNDLED_HTTP_PARSER=OFF` to CMake.
```

Comment:
We should treat changes to CMake options as breaking changes and thus highlight this in a separate section

Reply:
Generally we have a section titled "Breaking API Changes" at the bottom.  CMake changes don't feel like they fit there, so I created a new section "Breaking CMake configuration changes" and slotted this in there.  I also moved the "breaking" stuff to the top.  We should be done breaking APIs, but if not, it should be the first thing people see in the release notes, IMO.